addons/subscription/data/subscription_demo.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" ?>
 <odoo>
         <record id="base.user_demo" model="res.users">
-            <field eval="[(4, ref('base.group_sale_salesman'))]" name="groups_id"/>
+            <field eval="[(4, ref('sales_team.group_sale_salesman'))]" name="groups_id"/>
```

Comment:
Probably use group_user everywhere in the subscription module. This module can be used to create documents at a given frequency. It is not limited to sales, even if it is probably its main use.